guides/v3.4.0/models/index.md

```diff
@@ -1,5 +1,19 @@
-Models are objects that represent the underlying data that your
-application presents to the user. Different apps will have very
+Ember developers have great options for how they handle data from back end APIs. 
+Ember itself works with any type of back end: REST, JSON API, GraphQL, or anything else.
+
+Many developers choose to use Ember Data, a powerful set of tools for formatting requests, serializing responses, and efficiently managing a local cache of data. 
+The Ember Data library is included by default in Ember apps. 
```

Comment:
this sounds a lot like the paragraph later in the document:

> Ember Data, included by default when you create a new application, is a
>  library that integrates tightly with Ember to make it easy to retrieve
>  models from your server as JSON, save updates back to the server, and
>  create new models in the browser.

Reply:
Good catch. I removed the redundant description.